nuspec/nuget/Cake.Yeoman.nuspec

```diff
@@ -12,6 +12,7 @@
     <projectUrl>https://github.com/cake-contrib/Cake.Yeoman</projectUrl>
     <iconUrl>https://cdn.jsdelivr.net/gh/cake-contrib/graphics@a5cf0f881c390650144b2243ae551d5b9f836196/png/cake-contrib-medium.png</iconUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <repository type="git" url="https://github.com/cake-contrib/Cake.Yeoman"/>
```

Comment:
From the link to the nuspec specs, the repository link should also include the `.git` part of the url. (Not that it is really important though, just wanted to mention it)

Reply:
A `.git` url is used as an example on the following page: https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#pack-target

Reply:
Thanks. Updated to include the `.git` part